src/main/java/com/stripe/net/RequestOptions.java

```diff
@@ -394,7 +394,7 @@ protected static String normalizeClientId(String clientId) {
   }
 
   protected static String normalizeStripeVersion(String stripeVersion) {
-    // null stripeVersions are considered "valid" and use Stripe.stripeVersion
+    // null stripeVersions are considered "valid" and use Stripe.apiVersion
```

Comment:
Shouldn't it be `Stripe.stripeVersion`? https://github.com/stripe/stripe-java/blob/fb280de8ed50a12a16bf3bee2e358db9e0034b46/src/main/java/com/stripe/Stripe.java#L30

Reply:
You are right. I assumed what was in master to be the source of truth - didnt check the actual property name